firefox-ios/Client/Frontend/Autofill/LoginListViewModel.swift

```diff
@@ -0,0 +1,77 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import SwiftUI
```

Comment:
SwiftUI import is not needed here (@MainActor, ObservableObject and some of the other ones commonly used ones in SwiftUI VMs are available in Foundation) 

Personally, I try and avoid SwiftUI imports in view model classes just so view stuff doesn't accidentally get put in there.